README.md

```diff
@@ -4,6 +4,8 @@
 
 Tableau Python Server (TabPy) is part of Tableau's expanding range of extensibility options. These include [R execution](http://onlinehelp.tableau.com/current/pro/desktop/en-us/help.html#r_connection_manage.html) via the calculation editor interface, along with the web data connector SDK, the JavaScript API, the REST API, the Tableau Data Extract API, and Tableau Document API. For details, see the [Tableau Developer Portal](https://community.tableau.com/community/developers).
 
+For all not code related questions (installation, deployement, connections, Python issues, etc.) and requests use [Server & Online Administration Forum](https://community.tableau.com/community/forums/server-administration) on [Tableau COmmutity](https://community.tableau.com).
```

Comment:
+For all not code related questions

should be 
+For all non code related questions